src/app/features/reporter/reporter-dialog.component.html

```diff
@@ -42,29 +39,121 @@
+		<div class="col-md-12 mt-4 mb-4">
+			<h4 class="mb-4">Upload Results</h4>
+			<div>
+				<span>Total of Test Runs to upload:</span><span class="result-number"> {{totalTestsRun}}</span>
+			</div>
+			<div>
+				<span>Total of Test Runs uploaded:</span>
+				<span class="result-number"> {{testsRun['passed'] + testsRun['failed']}} (Pass {{testsRun['passed']}}, Failed {{testsRun['failed']}})</span>
+			</div>
+			<div *ngIf="testsRun['NotUpdated'] > 0">
+				<span>Number of Test Runs that exist in Jama but are not updated due to an error:</span>
+				<span class="result-number"> {{testsRun['NotUpdated']}}</span>
+			</div>
+			<div *ngIf="testsRun['NotExistsJamaInFiles'] > 0">
+				<span>Number of Test Runs that does not exist in Test Cycle in Jama:</span>
+				<span class="result-number"> {{testsRun['NotExistsJamaInFiles']}}</span>
+			</div>
+		</div>
+		<div class="col-md-12 mt-4" *ngIf="areResultsWrong()">
+			<h4>Wrong uploads</h4>
+			<div *ngIf="testsWrong['failed'].length > 0">
+				<h5 class="text-secondary">Test Runs with Failed status</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['failed']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['NotUpdated'].length > 0">
+				<h5 class="text-secondary">Test Runs that exist in Jama but are not updated due to an error</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['NotUpdated']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['NotExistsJamaInFiles'].length > 0">
+				<h5 class="text-secondary">Test Runs that does not exist in Test Cycle in Jama</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['NotExistsJamaInFiles']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div *ngIf="testsWrong['FileNotInJama'].length > 0">
+				<h5 class="text-secondary">Number of Test Runs that are pending in the selected Test Cycle</h5>
+				<table class="table table-sm table-striped">
+					<thead>
+						<tr>
+							<th scope="col">Test</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr *ngFor="let test of testsWrong['FileNotInJama']">
+							<td>{{test.name}}</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+		</div>
+	</div>
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" [disabled]="!this.selectedProjectId" (click)="doUpdateTestCase()">Update Test Case</button>
+	<div *ngIf="totalTestsRun > 0" class="progress allure-progress-bar" style="height: 20px;">
```

Comment:
Can you please use the standard progress-bar offered by systelab-dialog-header???

<systelab-dialog-header #header [withProgressBar]="true" (close)="close();">Progress Bar</systelab-dialog-header>